rubygem-smart_proxy_dns_route53/rubygem-smart_proxy_dns_route53.spec

```diff
@@ -0,0 +1,96 @@
+# Generated from smart_proxy_dns_route53-2.0.0.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name smart_proxy_dns_route53
+%global plugin_name dns_route53
+
+%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+Name: rubygem-%{gem_name}
+Version: 2.0.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Route 53 DNS provider plugin for Foreman's smart proxy
+Group: Applications/Internet
+License: GPLv3
+URL: https://github.com/theforeman/smart_proxy_dns_route53
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman-proxy >= 1.11
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+%else
+Requires: ruby(release)
+%endif
+Requires: ruby
+Requires: ruby(rubygems)
+Requires: rubygem(route53)
+%if 0%{?rhel} == 6
+BuildRequires: ruby(abi)
+%else
+BuildRequires: ruby(release)
+%endif
+BuildRequires: ruby
+BuildRequires: rubygems-devel
+BuildArch: noarch
+Provides: rubygem(%{gem_name}) = %{version}
+Provides: foreman-proxy-plugin-%{plugin_name}
+
+%description
+Route 53 DNS provider plugin for Foreman's smart proxy.
+
+
+%package doc
+Summary: Documentation for %{name}
+Group: Documentation
+Requires: %{name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+gem unpack %{SOURCE0}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build %{gem_name}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+# bundler file
+mkdir -p %{buildroot}%{foreman_proxy_bundlerd_dir}
+mv %{buildroot}%{gem_instdir}/bundler.d/%{plugin_name}.rb \
+   %{buildroot}%{foreman_proxy_bundlerd_dir}
+
+# sample config
+mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
+mv %{buildroot}%{gem_instdir}/settings.d/%{plugin_name}.yml.example \
```

Comment:
What I think I'm doing to do here is to copy any .yml  or .yml.example on `/config`to `foreman_proxy_settingsd_dir`. That sounds better than having both config/ and settings.d/ directories storing the same kind of files.